src/com/facebook/buck/cli/ProjectCommand.java

```diff
@@ -307,18 +308,31 @@ public boolean isExperimentalIntelliJProjectGenerationEnabled() {
     return experimentalIntelliJProjectGenerationEnabled;
   }
 
-  public List<String> getArgumentsFormattedAsBuildTargets(BuckConfig buckConfig) {
-    return getCommandLineBuildTargetNormalizer(buckConfig).normalizeAll(getArguments());
-  }
-
   @Override
   public int runWithoutHelp(CommandRunnerParams params) throws IOException, InterruptedException {
     if (getIde(params.getBuckConfig()) == ProjectCommand.Ide.XCODE) {
       checkForAndKillXcodeIfRunning(params, getIdePrompt(params.getBuckConfig()));
     }
 
-    ImmutableSet<BuildTarget> passedInTargetsSet = getBuildTargets(
-        getArgumentsFormattedAsBuildTargets(params.getBuckConfig()));
+    Pair<ImmutableSet<BuildTarget>, TargetGraph> result = null;
```

Comment:
Also, this doesn't compile.  On line 411 below we have `int result`, which introduces a duplicate variable name.